README.md

```diff
@@ -140,17 +140,17 @@ To connect with another player, one needs to configure a file named  `peers.json
 
 This file should be placed in a directory named after the cardano network used under one's home directory:
 
-* `hychess --network Preview` :point_right: `~/.config/hydra-node/preview/`
-* `hychess --network Preprod` :point_right: `~/.config/hydra-node/preprod/`
+* `hychess --network Preview` :point_right: `~/.config/hydra-chess/preview/`
+* `hychess --network Preprod` :point_right: `~/.config/hydra-chess/preprod/`
```

Comment:
Would it be possible to keep the config and cache in the xdg dirs for `hydra-chess` instead?

i.e. use the project name as a qualifier to not conflict with any potential forks of this easily

BTW: Love the usage of XDG directories :heart:

Reply:
Hmm, yes I see your point. I was afraid of mixing up a whole lot of different configuration, executables, data files in a single directory without much control and the possibility of conflicts _there_. How about a top-level directory `hydra-chess` with subdirectories for components, eg. :
```
~/.config/hydra-chess/preview/hydra-node/xxx
~/.config/hydra-chess/preview/cardano-node/xxx
...
```
?